packages/service/lib/start.js

```diff
@@ -61,7 +61,13 @@ async function buildServer (options, app) {
     }
     fastifyOptions.genReqId = function (req) { return randomUUID() }
     const root = fastify(fastifyOptions)
-    root.decorate('platformatic', { configManager, config, bus: new Bus(options.id) })
+    root.decorate('platformatic', {
```

Comment:
It looks like `currentService`, `bus` and `meta` properties make sense only if the service is in the runtime context. What happens when user runs a standalone plt service app?

Reply:
About the `bus` and `meta`, they will be available but nobody else will use them so no good but no harm as well.
I'm not really familiar with a standalone mode, but will the `options.id` property defined or not?

Reply:
Each of the platformatic stackables: `service`, `db` and `composer` (or any other custom stackable) can be run outside of the `runtime` context. Before, each of the stackable were written in the way that it don't know anything about the runtime.

If you want to add some features to the stackables that make sense only when they are executed in the runtime, please be careful. Ideally, I would add a property like `isRuntimeContext` that would be set to `true` by the runtime and enable all of the runtime specific logic only if it's set.


Something like:
```js
const context = { configManager, config }
if (isRuntimeContext) {
  context.currentService = options.id,
  context.bus = new Bus(options.id)
  context.meta = new Bus(options.id)
}
root.decorate('platformatic', context)
```

This will be needed not only for this place, but everywhere where we will define a runtime specific logic.

cc @mcollina

Reply:
Since the service should be agnostic about the runtime, I would say that bus and meta should always stay there, otherwise we will have to propagate `isRuntimeContext` as well to have the service know about its context as well.

The only doubt is about `currentService`. I checked and it is currently `undefined` when outside of a runtime. I'll mark as optional in the typings. Is this ok?

Reply:
Initializing properties with undefined does not make the service agnostic. If tomorrow you add some logic to the `Bus` class that would fail if `options.id` is `undefiend` this will not work anymore.

Reply:
Makes, sense. Ok, I will make the whole three options disappear.

Reply:
The three new options are not always present as you asked for.
There is also a new `isRuntimeContext` value now.